examples/tsdx/src/index.ts

```diff
@@ -0,0 +1,8 @@
+export { pay } from './pay';
+
```

Comment:
This work is unrelated to Web Payments SDK V1 upgrade. Can we remove it from this PR, and do this example under a separate PR.

Reply:
this is an implicit integration test that the types export properly and that consuming projects can build properly which is why I included it